ristretto/snapcraft.yaml

```diff
@@ -19,10 +19,15 @@ parts:
     plugin: autotools
     configflags:
       - --enable-maintainer-mode  # so src/marshal.h gets generated
-  env:
-    plugin: copy
-    files:
-      wrapper: usr/bin/ristretto.wrapper
+    build-packages:
```

Comment:
Did you have to move the `build-packages` definition? I always thought of it as something that's a global setting. Happy to move it though.